test/python/test_wrapper_helpers.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=invalid-name
+
+"""Tests for the wrapper functionality."""
+
+import unittest
+
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+from qiskit.wrapper import execute
+from qiskit.backends.local import QasmSimulatorCpp
+from .common import QiskitTestCase
+
+try:
+    cpp_backend = QasmSimulatorCpp()
+except FileNotFoundError:
+    _skip_cpp = True
+else:
+    _skip_cpp = False
+
+
+class TestWrapperHelpers(QiskitTestCase):
+    """Suite for testing `compile`, `execute` and other execution helpers."""
+
+    def setUp(self):
+        q = QuantumRegister(3)
+        c = ClassicalRegister(3)
+        self.circuit = QuantumCircuit(q, c)
+        self.circuit.ccx(q[0], q[1], q[2])
+        self.circuit.measure(q, c)
+
+    def test_local_execute_and_get_ran_qasm(self):
```

Comment:
Why create a new test file? Can you include in the the already existing `test_wrapper.py`?

Reply:
I did it because `test_wrapper.py` seems to be testing the functions related to `register()` and adding this functionality was somehow polluting the test case.

Reply:
Yeah but it should not just test `register`. As the name suggests, it should test the wrapper, which has several components (available_backends, register, execute). 
Our `test_wrapper.py` needs to get more complete and this is a good addition to it.